en/docs/ipaas/micro-integrator/developing-integrations-with-integration-studio.md

```diff
@@ -0,0 +1,106 @@
+
+## Integration Project Directory Structure
+
+Integration Project is a multi module maven project. It can contain multiple modules with WSO2 Synpase configurations
+and composite application module to export all the configurations as a single deployable artifact.
+Addition to the above OpenAPI definition file and libraries can be added optionally. OpenAPI definition can be added to
+the any location in the project and should be define it’s path when creating a component. Java libraries can be added to
+the directory created in the project root location with name libs.
+
+## OpenAPI support
+
+An [OpenAPI definition](https://spec.openapis.org/oas/v3.1.0#openapi-specification) can be developed to define your REST
+API. When a REST API is developed with integration studio and deployed on Choreo, it allows you to give the OpenAPI
+definition to attach to your Integration as an API component. Attaching an OpenAPI definition allows developers and
+consumers to understand the API.
+
+When a REST API is developed in WSO2 Integration Studio, developers can add an OpenAPI definition to the project. During
+component creation, the path to the OpenAPI Specification (OAS) file can be specified.
+
+![Open API File](/assets/img/ipaas/integration-studio/openapi_file.png){.cInlineImage-half}
+
+## Adding third party libraries
+
+Adding third-party jars to WSO2 Micro Integrator can bring several benefits to the development and deployment of
+integration solutions. Third-party jars can provide additional functionality that is not available in the core Micro
+Integrator distribution, allowing for greater flexibility and customization of integration solutions. They can also be
+used to integrate Micro Integrator with existing systems, such as databases or other APIs, improving the overall
+integration capabilities. Third-party jars can also provide optimized implementations of common functions, improving the
+performance of Micro Integrator. By using third-party jars, developers can avoid having to re-implement commonly used
+functions, reducing development time and increasing efficiency.
+
+To include such third-party libraries, a new folder can be created with the name __libs__. During the component
+deployment, all the __jars__ in the __libs__ directory in the project path will be included in the MI runtime.
+
+![Libs Dir](/assets/img/ipaas/integration-studio/libs_dir.png){.cInlineImage-half}
+
+## Support Environment variables
+
+Environment variables are used in WSO2 Micro Integrator to improve the configuration management, security, portability,
+and manageability of the installation. By using environment variables, organizations can simplify the management and
+maintenance of their MI installations, and ensure that they can be quickly and easily updated as needed.
+
+The Choreo DevOps portal allows configuring the __Configuration and secrets__ of your deployment. For more about how to
```

Comment:
Suggestion:
"The Choreo DevOps Portal allows you to define configurations and secrets to a deployed integration (as well as other component types). For more information, see [Configurations and secrets](../../devops/devops-portal.md#configurations-and-secrets)."